src/platform/service.cpp

```diff
@@ -44,5 +46,12 @@ namespace Toolbox::Platform {
 
         return isRunning;
     }
+#elifdef TOOLBOX_PLATFORM_LINUX
+    Result<bool, BaseError> IsServiceRunning(std::string_view name) {
+      return true;
```

Comment:
Is there such a concept as background services in Linux? If so, we should implement the check here. Otherwise, it should return false instead. (We use this to check for Nahimic which crashes OpenGL applications on window resize when active)

Reply:
I was trying to figure out if there's an equivalent, maybe just searching all running processes? Processes don't have a window by default on Linux, so maybe that's the equivalent. For now I just committed returning false instead to squash that warning, since the problem it's trying to detect doesn't exist on linux, so the high level behavior is correct.

Reply:
We can tackle this in a future PR if you like, but I'd say the idea of a running process is pretty much the same conceptually. A service in Windows is essentially just a background program that typically auto-starts on OS boot and provides "services" to other programs or the user in some capacity. So for Linux it would make sense that some background processes serve the same purpose without the "service" naming fluff.

Reply:
Hmm yeah seems useful for a future PR. I'm guessing that we'll want to bring the platform branching up a level, because there's no point in checking if if a Linux is running Nahimic, but maybe there will be other Linux specific things to check for there.

Reply:
Further investigation of these abstractions and their overall importance can be investigated in a future PR for sure. This will be sufficient until that point in time 👍 

Only thing that needs to be done is have it return false in the meantime to avoid false positives in the program itself.